cloudferrylib/os/actions/cinder_database_manipulation.py

```diff
@@ -66,10 +67,12 @@
 ])
 
 RSYNC_CMD = (
-    'rsync --progress --append -a --no-owner --no-group'
+    'rsync -Ppt --inplace'
```

Comment:
Please use long options, those are way more readable, don't need to be deciphered through man